pkg/validation/strfmt/default.go

```diff
@@ -124,6 +124,48 @@ func IsEmail(str string) bool {
 	return e == nil && addr.Address != ""
 }
 
+const dns1123LabelFmt string = "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
```

Comment:
if we're copying/pasting from apimachinery validation, are we inheriting these issues:

https://github.com/kubernetes/kubernetes/issues/71140
* This requires things to be lowercase, which is more strict than the RFC

https://github.com/kubernetes/kubernetes/issues/79351#issuecomment-510493021
* This tolerates individual labels exceeding 63 chars, which is not as strict as the RFC

Should we fix those issues before adopting those formats here?

Reply:
We intend to publish OpenAPI the in-tree types using these formats.  So I think we're stuck inheriting the it all.

That said, we shouldn't using the RFC identifiers for formats that are not exactly compliant with the RFC.  I'll propose some alternative names.